resources.go

```diff
@@ -838,6 +839,15 @@ func Provider() tfbridge.ProviderInfo {
 			"azurerm_mysql_firewall_rule":        {Tok: azureResource(azureMySQL, "FirewallRule")},
 			"azurerm_mysql_server":               {Tok: azureResource(azureMySQL, "Server")},
 			"azurerm_mysql_virtual_network_rule": {Tok: azureResource(azureMySQL, "VirtualNetworkRule")},
+			"azurerm_mssql_database_vulnerability_assessment_rule_baseline": {
+				Tok: azureResource(azureMySQL, "DatabaseVulnerabilityAssessmentRuleBaseline"),
```

Comment:
This is not MySQL, this is MS SQL (same for resources below)

Reply:
Great spot!!!